bulbs/instant_articles/parser.py

```diff
@@ -85,6 +85,13 @@ def parse_twitter(tag):
             return {'twitter': {'blockquote': six.text_type(blockquote)}}
 
 
+def parse_twitter_video(tag):
+    if tag.name == 'div' and tag.attrs.get('data-type') == 'embed':
+        blockquote = tag.find('blockquote', class_='twitter-video')
```

Comment:
This could instead be moved into `parse_twitter` and just do a for loop:

```
for class_ in ['twitter-video', 'twitter-tweet']:
         blockquote = tag.find('blockquote', class_=class_)
         if blockquote:
              return {'twitter': {'blockquote': six.text_type(blockquote)}}
```